src/core/thread/dua_manager.cpp

```diff
@@ -184,6 +242,498 @@ otError DuaManager::Store(void)
     return Get<Settings>().SaveDadInfo(dadInfo);
 }
 
+void DuaManager::AddDomainUnicastAddress(void)
+{
+    mDuaState             = kToRegister;
+    mLastRegistrationTime = TimerMilli::GetNow();
+    Get<ThreadNetif>().AddUnicastAddress(mDomainUnicastAddress);
+}
+
+void DuaManager::RemoveDomainUnicastAddress(void)
+{
+    if (mDuaState == kRegistering && mIsDuaPending)
+    {
+        IgnoreError(Get<Coap::Coap>().AbortTransaction(&DuaManager::HandleDuaResponse, this));
```

Comment:
Instead of ignoring the failure silently, will logging the error help debugging?